index.md

```diff
@@ -12,6 +12,16 @@ related DPM functionality, as well as a Cray extension, _MPIX\_Comm\_rankpool_.
 
 [Download MPI Dynamic Process Management Whitepaper](https://github.com/PE-Cray/whitepapers/raw/master/mpt/mpi-spawn.doc)
 
+
+**MPICH ABI Compatibility Status for HPE Cray EX Systems**
+
+<p align="justify">
+This document describes ANL MPICH ABI Compatibility on HPE Cray EX systems and how use this capability.
```

Comment:
how use this capability -> how to use this capability

Reply:
Fixed.